app/services/organization_update_service.rb

```diff
@@ -42,17 +44,29 @@ def update_partner_flags(organization)
     private
 
     def valid?(organization, params)
-      return true if organization.partners.none?
-      fields_marked_for_disabling = FIELDS.select { |field| params[field] == false }
+      fields_marked_for_disabling = FIELDS.select { |field| params[field] == "false" }
```

Comment:
I wouldn't do string comparisons against `false` here - the parameters should be cleaned up before it gets to the service (so replace the string "false" with a boolean `false` beforehand).

Reply:
Hmmm... it seems like the service is being called from the controller directly.  From my understanding, changing the incoming JSON parameters to represent the boolean `false` instead of the string `"false"` from within the controller wouldn't be a good option right?

And if I change it from outside of the service I would need an additional test, correct?

Reply:
No, the controller is a fine place to clean up parameters. In fact, I consider passing the parameters object as is to be a code smell since there's a bunch of logic in that class related to browser -> server translation.